src/components/HomePage/URLSearchParamsParser.js

```diff
@@ -7,6 +7,10 @@ import { useSelector } from "react-redux";
 import useOffersSearcher from "./SearchResultsArea/SearchResultsWidget/useOffersSearcher";
 import { SearchResultsConstants } from "./SearchResultsArea/SearchResultsWidget/SearchResultsUtils";
 
+import { ensureArray } from "../../utils";
+
+// TODO: should write tests for this
```

Comment:
I think they are done?

Reply:
They are, but in the `HomePage.spec.js` flle